source/dem/explicit_euler_integrator.cc

```diff
@@ -91,15 +91,27 @@ ExplicitEulerIntegrator<dim>::integrate(
 template <int dim>
 void
 ExplicitEulerIntegrator<dim>::integrate(
-  Particles::ParticleHandler<dim> & /* particle_handler */,
-  const Tensor<1, 3> & /* g */,
-  const double /* dt */,
-  std::vector<Tensor<1, 3>> & /* torque */,
-  std::vector<Tensor<1, 3>> & /* force */,
-  const std::vector<double> & /* MOI */,
+  Particles::ParticleHandler<dim> &particle_handler,
+  const Tensor<1, 3>              &g,
+  const double                     dt,
+  std::vector<Tensor<1, 3>>       &torque,
+  std::vector<Tensor<1, 3>>       &force,
+  const std::vector<double>       &MOI,
```

Comment:
This will be give warnings about unused parameters no? Why not keep the parameters commented out?

Reply:
At compilation? No because it can be used at line 111, otherwise, I'm not sure what you mean

Reply:
sorry about that.
my mistkae